C/tasks/north/sending_process/include/north_filter_pipeline.h

```diff
@@ -0,0 +1,27 @@
+#ifndef _NORTH_FILTER_PIPELINE_H
+#define _NORTH_FILTER_PIPELINE_H
+/*
+ * FogLAMP filter pipeline class for sending process
+ *
+ * Copyright (c) 2019 Dianomic Systems
+ *
+ * Released under the Apache 2.0 Licence
+ *
+ * Author: Amandeep Singh Arora
+ */
+
+#include <filter_pipeline.h>
+
+// NorthFilterPipeline class
```

Comment:
Can we please get into the habit of creating doxygen compatible comments for classes and methods